src/main/java/EncryptionAlgorithms/DoubleEncryption.java

```diff
@@ -14,17 +15,17 @@ public DoubleEncryption(EncryptionAlgorithm firstAlgorithm, EncryptionAlgorithm
         super("DoubleEncryption");
         this.algorithm1 = firstAlgorithm;
         this.algorithm2 = secondAlgorithm;
-        this.secondKey = (new Key("DoubleEncryption")).getKey();
+        this.secondKey = (new Key()).getKey();
     }
 
     @Override
-    public String encryptFile(String stringToEncrypt, int encryptionKey) throws IOException {
+    public String encryptFile(String stringToEncrypt, int encryptionKey) throws IOException, NoSuchElementException, ClassCastException {
```

Comment:
you not encrypt file here - change function name to just encrypt or encryptString etc.
*** in All Algorithems ***